src/Dapr.Workflow/DaprWorkflowClient.cs

```diff
@@ -70,9 +70,9 @@ public DaprWorkflowClient(DurableTaskClient innerClient)
         /// <param name="instanceId">The unique ID of the workflow instance to fetch.</param>
         /// <param name="getInputsAndOutputs">
         /// Specify <c>true</c> to fetch the workflow instance's inputs, outputs, and custom status, or <c>false</c> to
-        /// omit them. Defaults to false.
+        /// omit them. Defaults to true.
         /// </param>
-        public async Task<WorkflowState> GetWorkflowStateAsync(string instanceId, bool getInputsAndOutputs = false)
+        public async Task<WorkflowState> GetWorkflowStateAsync(string instanceId, bool getInputsAndOutputs = true)
```

Comment:
Just checking, but I assume this doesn't change the return payload in some way that isn't backwards compatible? We'd just get additional info in it, yeah?

Reply:
Thanks for checking. Yes, this change is backwards compatible. The only difference is that we add new data which was previously left as null.